locales/en/index.yml

```diff
@@ -1897,6 +1897,7 @@ services:
 serviceDetail:
   fiscalCode: "Organization's fiscal code"
   fiscalCodeAccessibility: "Organization's fiscal code"
+  fiscalCodeAccessibilityCopy: "Copia codice fiscale Ente"
```

Comment:
@thisisjp translation needed ❤️

Reply:
What do you think about using "tax code" instead "fiscal code" for the organization?

Reply:
I agree, it shoul be tax code

Reply:
With @thisisjp, we agreed to use this translation: `Copy organization’s fiscal code`.